docs/source/noise.rst

```diff
@@ -1044,33 +1038,43 @@ probability matrix directly from a QPU.
 
 **On a perfect quantum computer**
 
-.. code:: python
+.. testcode:: readout-noise
 
-    estimate_assignment_probs(0, 1000, qc)
+    print(estimate_assignment_probs(0, 1000, qc))
 
-.. parsed-literal::
+.. testoutput:: readout-noise
 
-    array([[ 1.,  0.],
-           [ 0.,  1.]])
+    [[1. 0.]
+     [0. 1.]]
 
 **On an imperfect quantum computer**
 
-.. code:: python
+.. testcode:: readout-noise
 
     qc.qam.random_seed = None
     header0 = Program().define_noisy_readout(0, .85, .95)
     header1 = Program().define_noisy_readout(1, .8, .9)
     header2 = Program().define_noisy_readout(2, .9, .85)
 
-    ap0 = estimate_assignment_probs(0, 100000, qc, header0)
-    ap1 = estimate_assignment_probs(1, 100000, qc, header1)
-    ap2 = estimate_assignment_probs(2, 100000, qc, header2)
+    ap0 = estimate_assignment_probs(0, 10000, qc, header0)
+    ap1 = estimate_assignment_probs(1, 10000, qc, header1)
+    ap2 = estimate_assignment_probs(2, 10000, qc, header2)
```

Comment:
I ended up lowering this because it was running into [this issue](https://github.com/rigetti/qcs-sdk-rust/issues/332), but I think it works out because this test already takes a long time to run.